gwy/templates/main.go.tmpl

```diff
@@ -234,10 +258,23 @@ func SetupMux(ctx context.Context, cfg proxyConfig) *http.ServeMux {
 
 	prefix := sanitizeApiPrefix(cfg.apiPrefix)
 	logrus.Infof("API prefix is: %s", prefix)
-	mux.Handle(prefix, handlers.CustomLoggingHandler(os.Stdout, http.StripPrefix(prefix[:len(prefix)-1], allowCors(cfg, gwmux)), formatter))
+	mux.Handle(prefix, handlers.CustomLoggingHandler(os.Stdout, http.StripPrefix(prefix[:len(prefix)-1], addRespHeaders(cfg, gwmux)), formatter))
 	return mux
 }
 
+// bindEnvVars will bind all env vars that starts with "prefix_secondPrefix"
+func bindEnvVars(viper *viper.Viper, r *strings.Replacer, prefix string, secondPrefix string) {
```

Comment:
I'd combine prefix and secondPrefix in the API - i.e. do the concat as a caller, it'll be easier.

Reply:
good call

Reply:
actually I need to to remove use both prefix for identifying the keys but than trim only the first prefix, so I think it be better to leave it as is.